src/cascadia/TerminalSettingsEditor/ColorSchemes.cpp

```diff
@@ -51,6 +51,17 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         L"Tango Light"
     };
 
+    void ColorSchemesPageNavigationState::RenameColorScheme(hstring oldName, hstring newName)
+    {
+        _Settings.RenameColorScheme(oldName, newName);
+    }
+
+    void ColorSchemesPageNavigationState::DeleteColorScheme(hstring name)
+    {
+        // This ensures that the JSON is updated with "Campbell", because the color scheme was deleted
+        _Settings.RenameColorScheme(name, L"Campbell");
```

Comment:
* * * NOT COMFORTABLE WITH THIS * * *
even momentarily, the settings model will think that _this color scheme_ that we're about to delete is campbell.